src/lib/campus-bounds.js

```diff
@@ -3,4 +3,6 @@ export default {
   west: 4.366779,
   south: 51.988050,
   east: 4.388487,
+  sw: [4.332918, 51.969528], // Southwest bound
```

Comment:
I think you can name the variables `southwest` and `northeast` instead of using a comment; this makes it more clear when you use them.